lib/gitlab/asciidoc.rb

```diff
@@ -0,0 +1,61 @@
+require 'asciidoctor'
+require 'html/pipeline'
+
+module Gitlab
+  # Parser/renderer for the AsciiDoc format that uses Asciidoctor and filters
+  # the resulting HTML through HTML pipeline filters.
+  module Asciidoc
+
+    # Provide autoload paths for filters to prevent a circular dependency error
+    autoload :RelativeLinkFilter, 'gitlab/markdown/relative_link_filter'
+
+    DEFAULT_ADOC_ATTRS = [
+      'showtitle', 'idprefix=user-content-', 'idseparator=-', 'env=github',
+      'env-github', 'env=gitlab', 'env-gitlab',
```

Comment:
The `env=github` and `env-github` attributes should be removed. We should only use `env=gitlab` and `env-gitlab` attributes.